website/docs/components/stepper/list/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,70 @@
+## Usage
+
+### When to use
+
+- In onboarding, account creation, or initial setups that are broken up into multiple steps.
+- To communicate what has and has not been completed in a multi-step flow.
+- For progress that does not require a user to remain on the current page to complete.
+
+### When not to use
+
+- When the user needs to remain within the step to complete it, use the [Stepper Nav](/components/stepper/nav).
+- For simple forms or when progress doesn't need to be tracked, use structured page content.
+
+## Content
+
+The Stepper List includes a title, an optional description, and a generic slot for content.
+
+### Title
+
+Titles provide an overview to help users understand what tasks need to be completed and their purpose within the flow. Titles are required and should be brief.
+
+### Description
+
+Descriptions are optional. They can be used to provide additional brief context. If used, we recommend providing a description for all steps.
+
+!!! Do
+
+Ensure inclusion of descriptions is consistent.
+
+![A stepper with descriptions for all four steps.](/assets/components/stepper/list/stepper-list-description-do.png)
+
+!!!
+
+!!! Dont
+
+Don't mix and match inclusion of descriptions.
+
+![A stepper where steps one and four have description but steps two and three only have titles.](/assets/components/stepper/list/stepper-list-description-dont.png)
+
+!!!
+
+## Progressive disclosure
+
+We recommend showing the contents of only the **in-progress** step while hiding the contents of other steps. This narrows the focus to only the current step and ensures steps are completed in order. However, this isn’t always feasible, and sometimes context from the previous step is needed for the user to complete the next step. Use your best judgment.
+
+!!! Do 
+
+Only show one call to action at a time.
```

Comment:
"Call to action" refers to the buttons or links present, not progressive disclosure. What about something like: 

Show the contents of one step at a time.